INSTALL.md

```diff
@@ -1,41 +1,7 @@
-NOTICE
-======
-
-Since Version 5.0, CGAL is a header-only library it is not needed
-to build and install it. Usage of CGAL should thus simply amount to:
-
-``` {.bash}
-git clone https://github.com/CGAL/cgal.git /path/to/cgal.git
-cd /path/to/cgal.git/Triangulation_2/examples/Triangulation_2
-mkdir -p build/debug
-cd build/debug
-cmake -DCMAKE_BUILD_TYPE=Debug -DCGAL_DIR=/path/to/cgal.git
-make
-```
-
-in the case of the building of an example in debug mode.
-
-For more information head over to the [CGAL manual](https://doc.cgal.org/latest/Manual/general_intro.html).
-Note that this page describes the setting of CGAL as a sources release and, as such,
-files are organized in a slightly different way, see the [Layout of the CGAL Git Repository](README.md).
-
-
 Building a Program Using CGAL
 =============================
 
-To compile a program using CGAL, simply set `CGAL_DIR` to the location
-of the directory containing `CGALConfig.cmake` (for example the root 
-of the extracted source archive or the root of a git checkout).
-
-Here is an example of how to build in debug the examples from the 3D Triangulations package:
-
-``` {.bash}
- cd /path/to/cgal.git/Triangulation_3/examples/Triangulation_3
- mkdir -p build/debug
- cd build/debug
- cmake -DCGAL_DIR:PATH=/path/to/cgal.git ../..
- make
-```
+Since Version 5.0, CGAL is header-only, hence it does not create a library. It needs however to link to the external libraries GMP and MPRF. 
```

Comment:
MPRF->MPFR
That's too specific. Some CGAL programs don't need to link with those, and some need to link with other libraries (zlib, Qt, etc).